crates/litesvm/Cargo.toml

```diff
@@ -13,6 +13,7 @@ internal-test = []
 nodejs-internal = ["dep:qualifier_attr"]
 
 [dependencies]
+ansi_term.workspace = true
```

Comment:
Does it make sense to make it optional? And gate the method behind a feature?

Reply:
I thought about this but the impact on compile times is negligible and the ansi_term crate is probably never getting a new release so it should never break anything

Reply:
Fine for me then